gen/calc.py

```diff
@@ -356,13 +356,13 @@ def validate_os_type(os_type):
         'dcos_overlay_mtu': '1420',
         'dcos_overlay_enable': "true",
         'dcos_overlay_network': '{                      \
-            "vtep_subnet": "198.18.0.0/20",             \
+            "vtep_subnet": "44.128.0.0/20",             \
             "vtep_mac_oui": "70:B3:D5:00:00:00",        \
             "overlays": [                               \
               {                                         \
                 "name": "dcos",                         \
-                "subnet": "44.128.0.0/16",              \
-                "prefix": 26                            \
+                "subnet": "9.0.0.0/8",                  \
```

Comment:
Meh. These are the defaults. The number of people whose DC/OS will integrate with IBM are fewer than the numbers who will. Having an experience in which they do not need to drift from the original configuration simplifies the deployments that can handle it.

I don't know who can answer the squishy question of "bad citizenship" - Maybe @spahl